R/post_ts.R

```diff
@@ -76,7 +74,7 @@ post_ts = function(files, on_exists=c("stop", "skip", "replace", "merge","archiv
             message("deleting timeseries data before replacement: ", ts_id)
             message("posting file to site ", ts_path$site_name, ", timeseries ", ts_path$ts_name)
           }
-          item_replace_files(ts_id, files = basename(files[i]), all=FALSE)
+          item_replace_files(ts_id, files = files[i], all=FALSE)
```

Comment:
and actually, basename doesn't remove the extension. what was the reasoning here?